packages/features/ee/teams/components/MemberInvitationModal.tsx

```diff
@@ -170,6 +170,7 @@ export default function MemberInvitationModal(props: MemberInvitationModalProps)
   const resetFields = () => {
     newMemberFormMethods.reset();
     newMemberFormMethods.setValue("emailOrUsername", "");
+    newMemberFormMethods.setValue("role", options[0].value);
```

Comment:
this is to fix a bug i encountered, notice the second time where the `invite as` default value is set to `member`, but the user ends up being invited as an `admin`. It seems like the previous value is persisting incorrectly.

https://www.loom.com/share/fa17f2ad127f41ff81b931a98963c7f2?sid=f06e1a9c-03c9-40c3-b090-3e53283efb4f